src/Parser/Core/Modules/Items/Legion/AntorusTheBurningThrone/EonarsCompassion.js

```diff
@@ -83,6 +83,14 @@ class EonarsCompassion extends Analyzer {
     }
   }
 
+  //if the regular proc procs again while it's already active
+  on_byPlayer_refreshbuff(event) {
+    const spellId = event.ability.guid;
+    if (spellId === SPELLS.EONARS_COMPASSION_PROCBUFF.id || this.triggerBuffs.includes(spellId)) {
```

Comment:
isn't triggerBuffs the pantheon buffs? I'd assume it should just look like `on_byPlayer_applybuff(event)`

Reply:
Fixed heh